runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOrePriceOverlay.java

```diff
@@ -0,0 +1,145 @@
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.motherlode;
+
+import java.awt.*;
+import javax.inject.Inject;
+import net.runelite.client.ui.overlay.OverlayPanel;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.TitleComponent;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.util.QuantityFormatter;
+
+public class MotherlodeOrePriceOverlay extends OverlayPanel
+{
+	private final MotherlodePlugin plugin;
+	private final MotherlodeSession motherlodeSession;
+	private final MotherlodeConfig config;
+
+	@Inject
+	MotherlodeOrePriceOverlay(MotherlodePlugin plugin, MotherlodeSession motherlodeSession, MotherlodeConfig config)
+	{
+		setPosition(OverlayPosition.TOP_LEFT);
+		this.plugin = plugin;
+		this.motherlodeSession = motherlodeSession;
+		this.config = config;
+	}
+
+	@Inject
+	private ItemManager itemManager;
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+		if (!plugin.isInMlm() || !config.showOrePrices())
+		{
+			return null;
+		}
+
+		MotherlodeSession session = motherlodeSession;
+
+		int nuggetsFound = session.getNuggetsFound();
+		int coalFound = session.getCoalFound();
+		int goldFound = session.getGoldFound();
+		int mithrilFound = session.getMithrilFound();
+		int adamantiteFound = session.getAdamantiteFound();
+		int runiteFound = session.getRuniteFound();
+
+		// If no ores have even been collected, don't bother showing anything
+		if (nuggetsFound == 0 && coalFound == 0 && goldFound == 0 && mithrilFound == 0
+			&& adamantiteFound == 0 && runiteFound == 0)
+		{
+			return null;
+		}
+
+		panelComponent.getChildren().add(TitleComponent.builder().text("Estimated value").build());
+
+		int nuggetsGP = itemManager.getItemPrice(12012) * nuggetsFound;
```

Comment:
Use `ItemID` references instead of hardcoded ints.

Reply:
Changes made.